webui/react/src/pages/DesignKit.tsx

```diff
@@ -0,0 +1,1228 @@
+};
+
+const PaginationSection: React.FC = () => {
+  return (
+    <ComponentSection id="Pagination" title="Pagination">
+      <Card>
+        <p>
+          <code>{'<Pagination>'}</code> is the process of splitting the contents of a website, or
+          section of contents from a website, into discrete pages. This user interface design
+          pattern is used so users are not overwhelmed by a mass of data on one page. Page breaks
+          are automatically set.
+        </p>
+      </Card>
+      <Card title="Best practices">
+        <strong>Content considerations</strong>
+        <ul>
+          <li>Use ordinal numerals or letters of the alphabet.</li>
+          <li>
+            Indentify the current page in addition to the pages in immediate context/surrounding.
+          </li>
+        </ul>
+      </Card>
+      <Card title="Usage">
+        <strong>Pagination default</strong>
+        <Pagination total={500} />
+        <strong>Considerations</strong>
+        <ul>
+          <li>
+            Always give the user the option to navigate to the first & last page -- this helps with
+            sorts.
+          </li>
+          <li>
+            Provide the user with 2 pages before/after when navigating &apos;island&apos;
+            page-counts.
+          </li>
+          <li>Provide the user with the first 4 or last 4 pages of the page-range.</li>
+          <li>
+            Ensure the FocusTrap is set to the whole pagination component so that user doesn&apos;t
+            tabs in/out accidentally.
+          </li>
+        </ul>
+      </Card>
+    </ComponentSection>
+  );
+};
+
+const DataCardsSection: React.FC = () => {
+  const rps = resourcePools as unknown as ResourcePool[];
+
+  return (
+    <ComponentSection id="DataCards" title="DataCards">
+      <Card>
+        <p>
+          A DataCard (<code>{'<OverviewStats>'}</code>) contains additional metadata or actions.
+          This offers people a richer view into a file than the typical grid view.
+        </p>
+      </Card>
+      <Card title="Best practices">
+        <strong>Content considerations</strong>
+        <ul>
+          <li>Incorporate metadata that is relevant and useful in this particular view.</li>
+          <li>
+            Don&apos;t use a document card in views where someone is likely to be performing bulk
+            operations in files, or when the list may get very long. Specifically, if you&apos;re
+            showing all the items inside an actual folder, a card may be overkill because the
+            majority of the items in the folder may not have interesting metadata.
+          </li>
+          <li>
+            Don&apos;t use a document card if space is at a premium or you can&apos;t show relevant
+            and timely commands or metadata. Cards are useful because they can expose on-item
+            interactions like “Share” buttons or view counts.
+          </li>
+        </ul>
+      </Card>
+      <Card title="Usage">
```

Comment:
This one doesn't have a matching component. Also @michielpr The variation here doesn't seem to have any relation to the default? What's the actual component here?